src/lib/Libifl/pbsD_submit_resv.c

```diff
@@ -70,9 +70,64 @@ __pbs_submit_resv(int c, struct attropl *attrib, char *extend)
 	struct attropl *pal;
 	int rc;
 	char *ret;
+	char job_id_resv[PBS_MAXJOBNAME + 1];
+	int msvr = multi_svr_op(c);
 
-	for (pal = attrib; pal; pal = pal->next)
+	job_id_resv[0] = '\0';
+	for (pal = attrib; pal; pal = pal->next) {
 		pal->op = SET;		/* force operator to SET */
+		if (msvr) {
+			if (strcmp(pal->name, ATTR_convert) == 0 || strcmp(pal->name, ATTR_resv_job) == 0) {
+				strcpy(job_id_resv, pal->value);
```

Comment:
Can you please confirm that ATTR_convert  and ATTR_resv_job are mutually exclusive and that ATTR_convert's, or qmove's, value is actually a jobid? Please add a comment here to explain what this if condition does.

Reply:
I have added a detailed comment in the code. Please check.
In both the above cases the value is a jobid.